signalapp/YYImage
# Fix static WebP images for iOS14/Xcode 12.

### Description

When building `YYImage` using Xcode 12 and running the demo app on iOS 14, static WebP images are not showing/displayed as expected (found the issue after running Signal on iOS 14 using Xcode 12, stickers do not show).

### Testing Steps

- Open `YYImageDemo.xcodeproj` and run the demo app.
- Select `WebP Images`.
- Confirm both animated and static images are showing.

**EDIT:** Forgot to mention, also checked using Xcode 11.7 + iOS 13.7, didn't spot any issues in the demo app.

### Screenshots

Before:
![Simulator Screen Shot - iPhone 11 - 2020-09-28 at 21 07 44](https://user-images.githubusercontent.com/675798/94498305-317c7f00-01d0-11eb-867b-084bc9f2aa45.png)

After:
![Simulator Screen Shot - iPhone 11 - 2020-09-28 at 21 07 26](https://user-images.githubusercontent.com/675798/94498322-38a38d00-01d0-11eb-975f-97a94fd1edac.png)
